schemas/clinvar-study.tables.json

```diff
@@ -0,0 +1,3068 @@
+            "datatype": "string",
+            "array_of": true
+          }
+        ]
+      },
+      {
+        "name": "type_sample",
+        "columns": [
+          {
+            "name": "uid",
+            "datatype": "string"
+          },
+          {
+            "name": "sample_description",
+            "datatype": "string"
+          },
+          {
+            "name": "origin",
+            "datatype": "string"
+          },
+          {
+            "name": "ethnicity",
+            "datatype": "string"
+          },
+          {
+            "name": "geographic_origin",
+            "datatype": "string"
+          },
+          {
+            "name": "tissue",
+            "datatype": "string"
+          },
+          {
+            "name": "cell_line",
+            "datatype": "string"
+          },
+          {
+            "name": "species",
+            "datatype": "string"
+          },
+          {
+            "name": "age",
+            "datatype": "string"
+          },
+          {
+            "name": "strain",
+            "datatype": "string"
+          },
+          {
+            "name": "strain",
+            "datatype": "string"
+          },
+          {
+            "name": "affected_status",
+            "datatype": "string"
+          },
+          {
+            "name": "number_tested",
+            "datatype": "integer"
+          },
+          {
+            "name": "number_males",
+            "datatype": "integer"
+          },
+          {
+            "name": "number_females",
+            "datatype": "integer"
+          },
+          {
+            "name": "number_chr_tested",
+            "datatype": "integer"
+          },
+          {
+            "name": "gender",
+            "datatype": "string"
+          },
+          {
+            "name": "family_data",
+            "datatype": "string"
+          },
+          {
+            "name": "proband",
+            "datatype": "string"
+          },
+          {
+            "name": "indication",
+            "datatype": "string"
+          },
+          {
+            "name": "citation",
+            "datatype": "string",
+            "array_of": true
+          },
+          {
+            "name": "xref",
+            "datatype": "string",
+            "array_of": true
+          },
+          {
+            "name": "comment",
+            "datatype": "string",
+            "array_of": true
+          },
+          {
+            "name": "source_type",
+            "datatype": "string"
+          }
+        ]
+      },
+      {
+        "name": "sample_description",
+        "columns": [
+          {
+            "name": "uid",
+            "datatype": "string"
+          },
+          {
+            "name": "description",
+            "datatype": "string"
+          },
+          {
+            "name": "citation",
+            "datatype": "string"
+          }
+        ]
+      },
+      {
+        "name": "age",
```

Comment:
Ergh... this doesn't make much sense as its own table, but it is complex so I guess we should follow the pattern...

If we think it's worth special-casing, we could just inline these columns on the "donor" table (or whatever the equivalent is)